source/agora/consensus/Fee.d

```diff
@@ -388,7 +388,7 @@ public class FeeManager
 
             this.db.execute(
                 "REPLACE INTO accumulated_fees (fee, public_key) VALUES (?,?)",
-                this.accumulated_fees[stake.output.address], stake.output.address);
+                this.accumulated_fees[stake.output.address].tupleof[0], stake.output.address);
```

Comment:
Out of curiosity, what's the meaning of `tupleof` in this context?
I only found the meaning for that in  [this page](https://dlang.org/spec/struct.html#struct_instance_properties).